src/Spout/Common/Escaper/XLSX.php

```diff
@@ -42,9 +42,9 @@ protected function init()
     public function escape($string)
     {
         $escapedString = $this->escapeControlCharacters($string);
-        // @NOTE: Using ENT_NOQUOTES as only XML entities ('<', '>', '&') need to be encoded.
-        //        Single and double quotes can be left as is.
-        $escapedString = htmlspecialchars($escapedString, ENT_NOQUOTES, 'UTF-8');
+        // @NOTE: We must use ENT_QUOTES because single and double quotes may be used in spreadsheet names.
```

Comment:
Can you please change that note to something more global like this: "Using ENT_QUOTES as XML entities ('<', '>', '&') as well as double quotes (for values of XML attributes) need to be encoded." ?